src/ui/sass/_colors.scss

```diff
@@ -7,8 +7,21 @@ $color-brand-white: #fff !default;
 $color-text-primary: #212121 !default;
 $color-text-secondary: #757575 !default;
 
-$color-link-primary: $color-brand-primary !default;
+$color-link-primary: var(--color-brand-primary) !default;
```

Comment:
What are the mechanics of this kind of default? Does the SASS compiler replace all instances of "$color-link-primary" with "var(--color-brand-primary)"?

Reply:
yup yup!